### DISCUSSION TITLE
Does someone know why the changes of a merged Pull Request doesn't appear in the branch it has been merged into?

### DISCUSSION THREAD

**Discussion by emilseuor** [community/community#36924] [Pull Requests]
Hi,

Since last month a strange situation has been happening over and over with my Pull Requests at the company I work at, the Pull requests are approbed and merged into main, but the changes never go into main, main continues with the same old code before the pull requests are merged... 

What could be happening? It's only with my PRs so far that I have seen it... 

I have screenshots

**Comment by thiswallz** [comment]
try to checkout main locally, and make a git status command to check if your changes were merged, also check the way you are merging with main (squash, merge, rebase) some times it can lead to not seeing the commit hash.

**Comment by ossanna16** [comment]
Hi there @emilseuor and welcome to our community 😄. Thanks so much for asking a great question!
